beta/nncf/tensorflow/quantization/algorithm.py

```diff
@@ -247,5 +251,19 @@ def loss(self) -> CompressionLoss:
     def initialize(self, dataset=None, loss=None):
         self._initializer(self._model, dataset, loss)
 
+        # TODO(Evgeny Tsykunov) Add BNadaptation support for detection and segmentation models
```

Comment:
Please, resolve this via config files.

Reply:
Now the BN adapt runs in the case of `batchnorm_adaptation` parameters presence in config. This is in accordance with [docs ](https://github.com/openvinotoolkit/nncf/blob/develop/docs/compression_algorithms/Quantization.md)